Add support for static boxes in 'leveled' type (#15959)
luanti-org/luanti: doc/lua_api.md

* `paramtype2 = "leveled"`
    * Only valid for "nodebox" with 'type = "leveled"', and "plantlike_rooted".
        * Leveled nodebox:

>             * Nodebox `type` must be set to `"leveled"`
>             * The level of the top face of the nodebox is stored in `param2`.
>             * The other faces are defined by 'fixed = {}' like 'type = "fixed"'
>               nodeboxes.
>             * The nodebox height is (`param2` / 64) nodes.
>             * The maximum accepted value of `param2` is 127.
>             * Boxes in '`leveled_fixed = {}`' will never change.

Reviewer: clarity
```suggestion
            * Nodebox `type` must be set to `"leveled"`
            * The height of all boxes in the `fixed` table will be set to (`param2` / 64).
            * The maximum accepted value of `param2` is 127.
            * Boxes in `leveled_fixed` will not be modified and used as-is.
```
